### ISSUE TITLE
mpw segfaults

### ISSUE THREAD

**Issue by siddharthist** [NixOS/nixpkgs#19280]
## Issue description

When I run `mpw`, it segfaults!

dmesg:

```
[137483.894693] mpw[22067]: segfault at 8 ip 000000000040727b sp 00007fff182659e0 error 4 in mpw[400000+a000]
[137582.376629] mpw[24286]: segfault at 0 ip 000000000040727b sp 00007ffc5dc462c0 error 4 in mpw[400000+a000]
```
### Steps to reproduce
1. Install mpw
2. Run it
3. Feed it some made up information
4. Watch it segfault
## Technical details
- System: 17.03.git.241de40 (Gorilla)
- Nixpkgs version: same

**Comment by dunn** [comment]
The same happens on 16.09.931.33c38c4 (Flounder), even when building from source.

**Comment by siddharthist** [comment]
This was fixed in e80a6dc62b33ca18ea6fbce777ee3b6abcf17ce8. Thanks, @pikajude!
